src/pages/training-and-education/force-multiplier-program.tsx

```diff
@@ -0,0 +1,234 @@
+import { GridContainer, Grid } from '@trussworks/react-uswds'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const ForceMultiplierProgram = () => {
+  return (
+    <>
+      <section className="banner usa-section bg-training text-white">
+        <div className="usa-prose grid-container">
+          <span className="text-uppercase text-ls-3 text-primary-light text-bold">
+            Training and Education
+          </span>
+          <h1 className="page-header">Become Digitally Fluent</h1>
+          <p className="usa-intro text-light font-heading-lg">
+            The Digital University Force Multiplier Program
+          </p>
+          <LinkTo
+            className="usa-button usa-button--big external-link--alt text-white text-no-underline"
+            href="http://digitalu.udemy.com">
+            Register
```

Comment:
I noticed this button doesn't have the same hover state as on the MVP site, not a huge deal but maybe worth just looking into?

Reply:
great catch! thank you!